Clase-02/git-basics.md

```diff
@@ -0,0 +1,850 @@
+
+Git es un **control de versiones distribuido** creado por [Linus Torvalds](https://www.cs.helsinki.fi/u/torvalds/) en 2005. 
+
+Al utilizarlo para gestionar proyectos, las personas que trabajan en equipo (y también las que trabajan de forma individual) pueden almacenar sus proyectos de manera remota (en la nube) y, a su vez, tener una copia local en la que trabajar.
+
+Guardar la información de forma remota posibilita que otras personas accedan a dicha información y se la descarguen, en caso de tener los permisos. A su vez, es en ese repositorio remoto donde cada miembro del equipo irá subiendo sus cambios para unificarlos con los del resto y, en caso de que algo le suceda a nuestra computadora, podemos tener la tranquilidad de que toda la información estára respaldada allí.
+
+## ¿Qué es GitHub?
+
+GitHub es un servidor remoto al cual se conecta nuestra computadora cada vez que subimos (pusheamos) o bajamos (clonamos o pulleamos) información, entre otras cosas. Así como existe [GitHub](https://github.com/), existen otros servidores: [GitLab](https://about.gitlab.com/), [Bitbucket](https://bitbucket.org/product/), [SourceForge](https://sourceforge.net/), etc.
+
+<div style="text-align:center">
+    <img src="git-basics-images/git-vs-github.webp" width="70%">
+    <br>
+    <em>
+        Imagen tomada de
+        <a href=https://blog.devmountain.com/git-vs-github-whats-the-difference>
+        Devmountain
+        </a>
+    </em>
+</div>
+
+Git me brinda una serie de comandos que me permiten interactuar con el servidor en donde alojaré mis archivos, pero el servidor que elija para esto es indistinto (por supuesto, siempre que sea un servidor compatible con git).
+
+<div style="text-align:center">
+    <img src="git-basics-images/git-servers.jpg" width="50%">
+    <br>
+    <em>
+        Meme generado con
+        <a href=https://imgflip.com/memegenerator>
+        Meme Generator
+        </a>
+    </em>
+</div>
+
+## ¿Cómo funciona git?
+
+En git, la información relacionada a nuestros archivos es manejada como una _copia instantánea_ de estos. Cada vez que se realiza un cambio en un proyecto, "se toma una foto" del estado de cada archivo en ese proyecto y se guarda la referencia. Para ser más eficiente, el sistema solo guarda nuevamente los archivos que se han modificado respecto de la última versión. Si un archivo no fue modificado, el sistema conserva la vieja versión que ya poseía almacenada.
+
+<div style="text-align:center">
+    <img src="git-basics-images/git-checkins-over-time.webp" width="70%">
+    <br>
+    <em>
+        Imagen tomada de
+        <a href=https://blog.softtek.com/es/git-c%C3%B3mo-poner-orden-en-el-caos>
+        Softtek
+        </a>
+    </em>
+</div>
+
+## Los estados de git
+
+Git tiene tres estados principales en los que se pueden encontrar los archivos: 
+
+- modificado (modified): significa que el archivo ha sido modificado pero todavía no se encuentra preparado para su confirmación, no ha sido añadido al índice
```

Comment:
Esta sugerencia y varias más en esta misma línea va a ser poner en itálicas los extranjerismos crudos cuando no aparecen castellanizados o cuando no están en código, pero fijate si te convence aceptarlos o no.

Reply:
Fer, ya acepté todas las otras. No me tomé (ni sé si voy a llegar a tomarme) el trabajo de volver a revisar todo para ver si en algún otro lado hay algún extranjerismo no puesto en itálicas (obvio que si me sobra tiempo mañana, lo hago). No me parece mal hacerlo así  De hecho, creo que es lo más correcto. Por ahí hubiese estado bueno charlar estas cuestiones de estilo antes (porque es un bajón que te hayas tenido que tomar todo este laburo y para ya hacerlo bien una vez y no tener que estar scrolleando comentario jaja). Pero lo tengo en cuenta para lá próxima así queda todo con el mismo estilo. Gracias por tomarte el trabajo de agregar todas las itálicas.